src/scheduler/analysis.py

```diff
@@ -487,3 +493,9 @@ def check_exceptions(self) -> bool:
             if plugin.check_exceptions():
                 return True
         return check_worker_exceptions([self.schedule_process, self.result_collector_process], 'Scheduler')
+
+
+def _fix_system_version(system_version: str | None) -> str:
```

Comment:
If I understand correctly this function is supposed to account for outdated database entrys that use the underscore instead of the hypen.

Shouldn't this be fixed in the database?